script/testing/oltpbench/constants.py

```diff
@@ -23,7 +23,10 @@
 OLTP_DEFAULT_BIN = os.path.join(OLTP_GIT_LOCAL_PATH, "oltpbenchmark")
 OLTP_DEFAULT_COMMAND_FLAGS = "--histograms  --create=true --load=true --execute=true -s 5"
 OLTP_DIR_CONFIG = os.path.join(OLTP_GIT_LOCAL_PATH, "config")
+OLTP_DIR_CUR = os.getcwd()
```

Comment:
I don't think this makes sense as a constant? I'm not entirely sure how this is used. Why couldn't we use OLTP_GIT_LOCAL_PATH?

Reply:
OLTP_DIR_CUR points to the file path where `run_oltpbench.py` is been called. In the `Jenkinsfile-nightly`, that is the `build` folder(which is in the subdirectory for each job, controlled by Jenkins). I didn't use `OLTP_GIT_LOCAL_PATH` because it is in the public `/tmp/` folder. There might be some concurrency problems if this script run multiple times in parallel.